packages/orchestration/test/supports.ts

```diff
@@ -30,13 +31,15 @@ export {
   makeFakeTransferBridge,
 } from '@agoric/vats/tools/fake-bridge.js';
 
-export const commonSetup = async (t: ExecutionContext<any>) => {
+// The common setup cannot support a durable zone because many of the fakes are not durable.
+// They were made before we had durable kinds (and thus don't take a zone or baggage).
+// To test durability in unit tests, test a particular entity with `relaxDurabilityRules: false`.
+// To test durability integrating multiple vats, use a RunUtils/bootstrap test.
```

Comment:
Should this comment be revised to become consistent with this PR?

Reply:
This is accurate. Integrating multiple vats still requires RunUtils (or higher fidelity). Unit tests won't.